sdk/kubeflow/trainer/utils/utils.py

```diff
@@ -317,6 +317,101 @@ def get_entrypoint_using_train_func(
     return container_command, [exec_script]
 
 
+def get_args_using_torchtune_config(
+    fine_tuning_config: types.TorchTuneConfig,
+) -> Tuple[List[str], List[str]]:
+    """
+    Get the Trainer args from the TorchTuneConfig.
+    """
+    args = []
+
+    # Override the dtype if it is provided.
+    if fine_tuning_config.dtype:
+        if fine_tuning_config.dtype not in [dtype.value for dtype in types.DataType]:
+            raise ValueError(
+                f"Unsupported dtype: {fine_tuning_config.dtype}. "
+                f"Supported dtypes are: {[dtype.value for dtype in types.DataType]}"
+            )
```

Comment:
I think, this can be simplified
```suggestion
    if not (fine_tuning_config.dtype, types.DataType):
      raise ValueError(f"Invalid dtype: {fine_tuning_config.dtype})
```